src/utils/addresses.ts

```diff
@@ -31,33 +34,27 @@ export const checkAddressValidity = (address: string) => {
   return match[0] === address && address
 }
 
-export const loadStoredAddressesInfoOfAccount = (username: string): AddressInfo[] => {
-  const storedAddressIndexes = localStorage.getItem(`${username}-address-info`)
+export const loadStoredAddressesIndexAndSettingsOfAccount = (username: string): AddressIndexAndSettings[] => {
+  const data = localStorage.getItem(`${username}-address-indexes-and-settings`)
```

Comment:
Same remark here. This will be tricky to migrate if we decide to store more metadata than index + settings one day.